src/main/java/app/mapper/AbstractMapper.java

```diff
@@ -0,0 +1,28 @@
+package app.mapper;
+
+import app.config.MapperConfig;
+import org.modelmapper.ModelMapper;
+
+public class AbstractMapper<D, E> implements Mapper<D, E> {
+
+    protected ModelMapper modelMapper;
+    protected Class<D> dtoClass;
+    protected Class<E> entityClass;
```

Comment:
Maybe these should be final? I don't think we'd want them changed as that wouldn't make much sense